core/src/main/scala/org/bykn/bosatsu/MainModule.scala

```diff
@@ -1146,6 +1149,21 @@ abstract class MainModule[IO[_]](implicit
       }
     }
 
+    case class Show(
+        inputs: Inputs.Runtime,
+        output: Option[Path],
+        errColor: Colorize
+    ) extends MainCommand("show") {
+
+      type Result = Output.ShowOutput
+
+      def run = withEC { implicit ec =>
+        for {
+          (packs, _) <- inputs.packMap(this, Nil, errColor)
```

Comment:
I think packMap is added predef, which we shouldn't include by default, I don't think. Maybe a flag?